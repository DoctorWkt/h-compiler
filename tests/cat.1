.oh '11/3/70''CAT(I)'
.ta 18 21
NAME	cat -- concatenate and print
.sp
SYNOPSIS	
.ul
cat
file1 ...
.sp
DESCRIPTION	
.ul
cat
reads each file in sequence and writes it on
	the standard output stream. Thus:
.sp
.ul
		cat file
.sp
	is about the easiest way to print a file. Also:
.sp
.ul
		cat filel file2 >file3
.sp
	is about the easiest way to concatenate files.
.sp
FILES	--
.sp
SEE ALSO	pr, cp
.sp
DIAGNOSTICS	If a file cannot be found it is ignored. If no
	files are given, "no files" is output.
.sp
BUGS	--
.sp
OWNER	ken, dmr
