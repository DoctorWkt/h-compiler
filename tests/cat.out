

11/3/70                                                    CAT(I)


NAME             cat -- concatenate and print

SYNOPSIS         _c_a_t file1 ...

DESCRIPTION      _c_a_t reads each file in sequence and writes it on
                 the standard output stream. Thus:

                    _c_a_t _f_i_l_e

                 is about the easiest way to print a file. Also:

                    _c_a_t _f_i_l_e_l _f_i_l_e_2 >_f_i_l_e_3

                 is about the easiest way to concatenate files.

FILES            --

SEE ALSO         pr, cp

DIAGNOSTICS      If a file cannot be found it is ignored. If no
                 files are given, "no files" is output.

BUGS             --

OWNER            ken, dmr




































